int64 id
int64 top_right
int64 top_left
int64 bottom_right
int64 bottom_left
int64 position
float64 distance